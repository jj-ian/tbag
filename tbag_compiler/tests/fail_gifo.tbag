room {}

room Closet { 
	name = "Closet";
}

room Bedroom {
	name = "Bedroom";
}

Closet <-> Bedroom;

start { Bedroom }

currentRoom == Closet {
	print("Currently in: ");
	print(currentRoom.name);
	print("\n");
	get_input_from_options("Bedroom", 1);
	->input
	print("\n");
        endgame;
}

