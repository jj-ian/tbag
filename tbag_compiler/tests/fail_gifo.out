Fatal error: exception Failure("get_input_from_options expects 
                      one or more string arguments")
